stacktrace_ext_test.go

```diff
@@ -37,22 +32,23 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-// _zapPackages are packages that we search for in the logging output to match a
-// zap stack frame. It is different from _zapStacktracePrefixes which  is only
-// intended to match on the function name, while this is on the full output
-// which includes filenames.
-var _zapPackages = []string{
-	"go.uber.org/zap.",
-	"go.uber.org/zap/zapcore.",
-}
-
 func TestStacktraceFiltersZapLog(t *testing.T) {
 	withLogger(t, func(logger *zap.Logger, out *bytes.Buffer) {
 		logger.Error("test log")
 		logger.Sugar().Error("sugar test log")
 
-		require.Contains(t, out.String(), "TestStacktraceFiltersZapLog", "Should not strip out non-zap import")
-		verifyNoZap(t, out.String())
+		frames := strings.Split(out.String(), "\n")
+
+		assert.True(t, len(frames) > 6)
+
+		// The logging function should be the first frame ...
```

Comment:
instead of testing partially, can we just test the entire result of `frames`?

Reply:
Sure, added.  I was worried it'd be too brittle.